File: 1938-minimum-operations-to-make-the-array-increasing/minimum-operations-to-make-the-array-increasing.py
class Solution:
    def minOperations(self, nums: List[int]) -> int:
        y=len(nums)
        c=0
        if(y<=1):
            return 0
        for i in range(y-1):
            if(nums[i]>=nums[i+1]):
                c=c+((nums[i]+1)-nums[i+1])
                nums[i+1]=nums[i]+1
        if(nums[y-2]>=nums[y-1]):
                c=c+((nums[y-2]+1)-nums[y-1])
        return c
        
        

File: 1938-minimum-operations-to-make-the-array-increasing/minimum-operations-to-make-the-array-increasing.java
class Solution {
    public int minOperations(int[] nums) {
        int c=0;
        int k=nums.length;
        if(k<=1){
            return 0;
        }
       for(int i=0;i<nums.length-1;i++){
          if(nums[i]>=nums[i+1]){
             c+=((nums[i]+1)-nums[i+1]);
             nums[i+1]=(nums[i]+1);
          }
       }
      
       if(nums[k-2]>=nums[k-1]){
             c+=((nums[k-2]+1)-nums[k-1]);
             nums[k-1]=c;
        }
       
       return c; 
    }
}